pulsar-broker/src/main/java/org/apache/pulsar/broker/web/PulsarWebResource.java

```diff
@@ -1166,7 +1178,7 @@ public void validateTopicOperation(TopicName topicName, TopicOperation operation
         }
     }
 
-    protected Void handleCommonRestAsyncException(AsyncResponse asyncResponse, Throwable ex) {
+    protected static Void handleCommonRestAsyncException(AsyncResponse asyncResponse, Throwable ex) {
```

Comment:
Sorry I didn't see that the return type is Void.

Please it to 'void'

Reply:
Could we use it in ``exceptionally`` to avoid unnecessary ``return null``?

Use ``Avoid`` return type
```java
//....
.exceptionally(ex -> {
     return handleCommonRestAsyncException(asyncResponse, ex);
})
```
Don't use ``Avoid`` return type
```java
//....
.exceptionally(ex -> {
     handleCommonRestAsyncException(asyncResponse, ex);
     return null;
})
```